sys/arduino/arduino_sketches.inc.mk

```diff
@@ -0,0 +1,31 @@
+# Compile together the Arduino sketches of the application
+# They are declared a as new module $(SKETCH_MODULE) in $(SKETCH_MOD_DIR)
```

Comment:
typo SKETCH_MOD_DIR -> SKETCH_MODULE_DIR

Reply:
Thanks, I missed this one.